src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityMenu.js

```diff
@@ -52,7 +54,7 @@ const AnnotationActivityMenu = ({ canDelete, canEdit, id, onDeleteConfirm, onEdi
                     'data-resin-feature': 'annotations',
                 }}
             >
-                {canEdit && (
+                {!shouldDisableModifyMenuItem && canEdit && (
```

Comment:
Are there any lint issues with this? Maybe just remove the menu item entirely and add it back later?

Reply:
I don't see any lint issues in the build that just ran